core/include/detray/definitions/detail/algorithms.hpp

```diff
@@ -18,7 +19,7 @@
 namespace detray::detail {
 
 /// @brief sequential (single thread) sort function
-template <class RandomIt>
+template <detray::ranges::random_access_iterator RandomIt>
```

Comment:
Question: How does this differ from `std::random_access_iterator`?

Reply:
That's a fair point. Initially, I wanted to decorate the detray iterators with iterator tags from thrust as well, which is why detray has its own implementation of these. At the moment though, the detray iterator tags and the std tags are synonymous